### PR TITLE
Add Scheduler Acceptance Tests [read commit message before merge]

### LINKED ISSUE #184: Add Tests for Scheduling
These tests should validate:
* Create
* Delete
* List
* List filtered by task definition.

### LINKED ISSUE #185: Update Scripts that support the Scheduling on CF
Update the CF AT scripts so that:
* they can create the scheduler service 
* configure the dataflow to use the scheduler service
* a user can configure the AT to use an existing service

### PR DESCRIPTION
**Pull request by cppwfs**
* Implemented infrastructure to execute Schedule AT's for CloudFoundry.
* Junit Schedule Tests are skipped  when scheduling is disabled, so it safe to run in environments where scheduling is disabled..
* Updated SCDF client version to 1.7.0.BUILD-SNAPSHOT
* Scripts by default do not configure dataflow to enable scheduling and do not setup the accompaning infrastructure.   The -se option must be used to setup the environment for scheduling.
* Stubbed out infrastructure to create and delete scheduling services  for k8's and local, but they print a warning message
* Added notes to the README
* need to add` export SCHEDULES_URL=<schedulerurl>` to environment props.

* Before merging this PR, a branch will need to be created for the current master.  Once that branch is created all bamboo 1.6.x based Acceptance Test Plans will need to pull from that branch.

Resolves #184
Resolves #185

### CODE REVIEW

**Comment by chrisjs** [comment]
per conversation, maybe to keep things consistent use "scheduler" or "schedules" across the board rather than a mix of "scheduler" and "schedules".

**Comment by cppwfs** [comment]
Unfortunately updating the version, does not work :-( (I tried)  Because the schedule tests rule to skip the tests still use `ScheduleOperations`.   
But I will still leave the property change in place

File: local/server/create.sh [hunk 1]
```diff
@@ -27,5 +27,9 @@ if [  -z "$skipperMode" ]; then
  APPLICATION_ARGS="$APPLICATION_ARGS --spring.cloud.deployer.local.workingDirectoriesRoot=$APP_LOG_PATH"
 fi
 
+if [ "$schedulesEnabled" ]; then
```
Comment (chrisjs) [inline local/server/create.sh]: similar to the k8s `create.sh` for consistency should this just be:

```APPLICATION_ARGS="$APPLICATION_ARGS --spring.cloud.dataflow.features.schedules-enabled=$SPRING_CLOUD_DATAFLOW_FEATURES_SCHEDULES_ENABLED
```

**Comment by chrisjs** [comment]
ok got it. LGTM - leaving for @ilayaperumalg for any feedback

**Comment by sabbyanandan** [comment]
We would have to decide what the next version should be and likewise, we will have to create a new branch to merge this PR.

File: cloudfoundry/init/env.properties [hunk 2]
```diff
@@ -21,3 +21,7 @@ LOG_SERVICE_NAME=
 CONFIG_SERVER_SERVICE_NAME=p-config-server
 CONFIG_SERVER_PLAN_NAME=trial
 TRUST_CERTS=api.run.pivotal.io
+SCHEDULES_SERVICE_NAME=scheduler-for-pcf
+SCHEDULES_PLAN_NAME=standard
+SCHEDULES_INSTANCE_NAME=atscheduler
+SCHEDULES_URL=https://scheduler.sys.needles.cf-app.com
```
Comment (ilayaperumalg) [inline cloudfoundry/init/env.properties]: Please remove the explicit names here. We can have them configured in the CI plan instead.

**Comment by ilayaperumalg** [comment]
LGTM

**Comment by cppwfs** [comment]
Rebased Retested.

**Comment by cppwfs** [comment]
oops.  Clicked wrong button

**Comment by chrisjs** [comment]
merged in f7f7aa

**Pull request closed without merging**